Group 3

olhanotolga opened the pull request:
Group 3:
Amit, Cindy, Christin, Feruj, Tomas, Sezai, Tony, Olha

ciamac-da reviewed (changes requested):
The logic works only for numbers without decimals, please also fix the code to get correct result for decimal numbers! For example if i try to write 149.1 instead of 80 I get Not enough notrs to give change instead of 0.9

olhanotolga commented:
@ciamac-da which file did you refer to?

In the second file, `nowMoreComplicated.js`, we changed the conditions!
We added cash that we have "in the store" (`cashier.desk`) with notes and their number available to us, and if you log it, you can see that there are really no €0.90 to give:
```
{
  '€500': 1,
  '€50': 2,
  '€10': 3,
  '€2': 3,
  '€1': 1,
  '€0.50': 1,
  '€0.02': 5,
  '€0.01': 20
}

```

I updated the files with a few additional logs.

Please let me know if I still misunderstood you!

(To be honest, I looked at it for a while thinking "why can't I give 9 cents change when I have 20 x 1cent notes..." But then I saw it was 90 cents...!)

ciamac-da commented:
Many Thanks for your clear explanation!:)
